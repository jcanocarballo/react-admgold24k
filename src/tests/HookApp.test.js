import '@testing-library/jest-dom';
import {shallow} from 'enzyme';
import { HookApp } from '../HookApp';

describe('Pruebas para el archivo HookApp.js', () => {
  test('Deberia mostrar el componente <HookApp /> correctamente', () => {    
    const wraper = shallow(<HookApp></HookApp>);
    expect(wraper).toMatchSnapshot();
  });
  
});