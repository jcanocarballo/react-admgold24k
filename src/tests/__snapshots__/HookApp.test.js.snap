// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas para el archivo HookApp.js Deberia mostrar el componente <HookApp /> correctamente 1`] = `
<div>
  <h1>
    Hook App
  </h1>
</div>
`;
